shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-scaling/src/test/resources/env/mysql/01-initdb.sql

```diff
@@ -23,3 +23,4 @@ CREATE DATABASE scaling_it_4;
 
 GRANT REPLICATION CLIENT, REPLICATION SLAVE, SELECT, INSERT, UPDATE, DELETE, INDEX ON *.* TO `test_user`@`%`;
 GRANT CREATE, DROP ON TABLE *.* TO test_user;
+ALTER USER `test_user`@`%` IDENTIFIED WITH mysql_native_password by 'Test@123';
```

Comment:
Is it required? (ALTER USER ... mysql_native_password)

Reply:
It's used for mysql 8 , the test_user default plugin is same as `default_authentication_plugin` config at my.cnf,  so need to override at here

Reply:
OK